Assets/Scripts/Models/Area/World.cs

```diff
@@ -478,6 +484,7 @@ private void TickEveryFrame(float deltaTime)
     {
         CharacterManager.Update(deltaTime);
         FurnitureManager.TickEveryFrame(deltaTime);
+        GameEventManager.Update(deltaTime);
         ShipManager.Update(deltaTime);
```

Comment:
@koosemose This system needs to get expanded. So here I just made it so that it worked exactly the same as in master. We should probably discuss this further in another issue.

In a way an event shouldn't have a repetition. There should be a story teller module that will have a list of possible events with information about the amount and conditions as to when the events can occur. At that point the GameEventManager could keep track of 2 loops, one slow to check for upcoming events, and maybe a fast one for the current events, if the event needs it. And once the event ends, it gets removed from the list. For example some sort of meteor shower could be on a fast update so that the animation is handled better, and at the same time could be in the slow update to check for when it will end or other less important things.